gc.c

```diff
@@ -958,14 +958,130 @@ static const char *obj_info(VALUE obj);
  * written by Kazutomo Yoshii <kazutomo@mcs.anl.gov>
```

Comment:
Could you modify this comment (it can be misleading), and write your name (e.g  `modified by ... [PR-LINK]`) if you want :).